webapp/src/ts/app.component.ts

```diff
@@ -563,6 +569,7 @@ export class AppComponent implements OnInit, AfterViewInit {
       .then(({ privacyPolicy, accepted }: any = {}) => {
         this.globalActions.setPrivacyPolicyAccepted(accepted);
         this.globalActions.setShowPrivacyPolicy(privacyPolicy);
+        this.showContent = true;
```

Comment:
I'm not a fan of setting the value here because it's a bit unexpected when calling a function called `checkPrivacyPolicy` that it would suddenly show content. I'd add a new `then` just below line 281, eg:

```
.then(() => this.startRecurringProcesses())
.then(() => this.showContent = true)
.catch(err ...)